gsrs-spring-legacy-indexer/src/main/java/ix/core/search/text/TextIndexer.java

```diff
@@ -1475,8 +1475,16 @@ public SearchResult search(GsrsRepository gsrsRepository,  SearchOptions options
     		} else {
     			try {
     				QueryParser parser = new IxQueryParser(FULL_TEXT_FIELD, indexerService.getIndexAnalyzer());
-    				query = parser.parse(qtext);
-                    /*System.out.println("query class: " + query.getClass().getName());*/
+    				String processedQtext = qtext.trim();
+    				//phrase query with wildcards 
+     				if( processedQtext.startsWith("\"") && processedQtext.endsWith("\"")
```

Comment:
Can we have a deeper explanation for what's going on here? And remove the extra commented out code? It's a little hard to follow as-is.

This looks like it will only address cases where the query does not have a field name? So something like 
```
root_names_name:"*OCT-2*"
```
Would still fail to be transformed into a space. Is there a way to deal with that?

Can we also take the basic intended logic and make it its own method and add a unit test for it?